server/chat-commands.js

```diff
@@ -609,6 +609,41 @@ const commands = {
 		`(in a PM) /invite [roomname] - Invites the player you're PMing to join the room [roomname].`,
 	],
 
+	tourexport: 'tournamentexport',
+	export: 'tournamentexport',
+	tournamentexport(target, room) {
```

Comment:
This should be a subcommand of the tournament command, not in `chat-commands`

Reply:
ok, can you run the tournament command after a tournament has ended?

Reply:
WIll come back to this, am so busy.